packages/babel-parser/src/parser/util.js

```diff
@@ -259,6 +259,22 @@ export default class UtilParser extends Tokenizer {
       this.raise(doubleProto, Errors.DuplicateProto);
     }
   }
+
+  /**
+   * Test if current token is a literal property name
+   * LiteralPropertyName:
```

Comment:
Should this also include `BigIntLiteral`?

Reply:
The production is extracted from the [spec](https://tc39.es/ecma262/#prod-LiteralPropertyName) where `BigIntLiteral` is covered by `NumericLiteral`. I am happy to add that if it helps.

Reply:
Nah the link is fine